debian/postinst

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Why bash? The maintscripts use no bash feature. But if you insist in using bash here, you should add bash as `Pre-Depends` in control.

Reply:
Ah, maybe you read [this comment](https://github.com/zabbix-deb/zabbix-check-spamassassin/pull/1/files#r563567123) and changed the shebang in the wrong file?

Reply:
I still don't think that such systems can exist. `apt-cache show bash` says: `Priority: required`

https://www.debian.org/doc/debian-policy/ch-archive.html#s-priorities


>required
>
>    Packages which are necessary for the proper functioning of the system (usually, this means that dpkg functionality depends on these packages). Removing a required package may cause your system to become totally broken and you may not even be able to use dpkg to put things back, so only do so if you know what you are doing.
>
>    Systems with only the required packages installed have at least enough functionality for the sysadmin to boot the system and install more software.